concent_api/core/payments/bankster.py

```diff
@@ -321,9 +313,14 @@ def settle_overdue_acceptances(
             transaction_type=TransactionType.FORCE,
         )
 
+        already_satisfied_claims = remove_duplicated_transactions_from_deposit_claims(
+            already_satisfied_claims,
+            list_of_forced_payments,
+        )
+
         (_amount_paid, amount_pending) = get_provider_payment_info(
             list_of_forced_payments=list_of_forced_payments,
-            list_of_payments=list_of_transactions,
+            already_satisfied_claims=already_satisfied_claims,
```

Comment:
:+1: done, about second part maybe you can update https://github.com/golemfactory/concent/issues/1018 which is not started yet.

Reply:
Right, I have just suggested that maybe before starting https://github.com/golemfactory/concent/issues/1018 (we marked this one as blocked until other issues related with UC5 are resolved) it might be good for you to review the issue description.

Reply:
I don't think updating the issue to exactly match the code is necessary. The gist of the issue is that the names are bad or went out of sync with specs and need to be improved.

There are low level details such as specific names that I found problematic and justification for why they should be renamed but that's only to make it clear what I found particularly bad about those names. that' specifically to make it easy for the implementer to get what I meant and apply it to the new names if they change.